store/store_test.js

```diff
@@ -9,20 +11,54 @@ let user = {
     password: '987456',
 }
 
+let shippingPrice = 2;
+
 Feature('store');
 
-Scenario('test something', ({ I, homePage, authPage, createAccountPage, myAccountPage }) => {
+Before(({ I }) => {
+    I.openStore();
+});
+
+xScenario('test something', ({ I, homePage, authPage, createAccountPage, myAccountPage }) => {
     I.openStore();
     homePage.clickSignIn();
-    authPage.fillEmail(Date.now() + '@test.com');
+    authPage.fillRegistrationEmail(Date.now() + '@test.com');
     authPage.clickCreateAccount();
     createAccountPage.submitNewUserFields(user);
     myAccountPage.verifyMyAccountPage();
 }).tag('auth');
 
-
 xScenario('grab', async ({ I }) => {
     I.amOnPage('http://automationpractice.com/index.php?id_product=1&controller=product');
     let price = await I.grabTextFrom({css:'#our_price_display'});
     console.log(price);
-}).tag('grab');
\ No newline at end of file
+}).tag('grab');
+
+Scenario('buy product', async ({ I, homePage, authPage, productPage, cartPage}) => {
+    homePage.clickSignIn();
+    authPage.login('felinarien23@gmail.com', 'chrisevans2311');
+    I.amOnPage('http://automationpractice.com/index.php?id_product=3&controller=product');
+
+    let productPrice = await productPage.getProductPrice();
+    let productPriceNumber = Number(productPrice.slice(1));
```

Comment:
move string to number conversion to `getProductPrice()` here and for all price methods below